pkg/testutils/fullstack.go

```diff
@@ -0,0 +1,213 @@
+package testutils
+
+import (
+	"context"
+	"crypto/ecdsa"
+	"database/sql"
+	"encoding/hex"
+	"net/http/httptest"
+	"testing"
+	"time"
+
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	"github.com/ethereum/go-ethereum/accounts/abi/bind/backends"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/crypto"
+	"github.com/stretchr/testify/require"
+	"github.com/textileio/go-tableland/internal/chains"
+	"github.com/textileio/go-tableland/internal/router"
+	"github.com/textileio/go-tableland/internal/system"
+	systemimpl "github.com/textileio/go-tableland/internal/system/impl"
+	"github.com/textileio/go-tableland/internal/tableland"
+	"github.com/textileio/go-tableland/internal/tableland/impl"
+	"github.com/textileio/go-tableland/pkg/client"
+	"github.com/textileio/go-tableland/pkg/eventprocessor/eventfeed"
+	efimpl "github.com/textileio/go-tableland/pkg/eventprocessor/eventfeed/impl"
+	epimpl "github.com/textileio/go-tableland/pkg/eventprocessor/impl"
+	executor "github.com/textileio/go-tableland/pkg/eventprocessor/impl/executor/impl"
+	nonceimpl "github.com/textileio/go-tableland/pkg/nonce/impl"
+	"github.com/textileio/go-tableland/pkg/parsing"
+	parserimpl "github.com/textileio/go-tableland/pkg/parsing/impl"
+	"github.com/textileio/go-tableland/pkg/sqlstore"
+	sqlstoreimpl "github.com/textileio/go-tableland/pkg/sqlstore/impl"
+	sqlstoreimplsystem "github.com/textileio/go-tableland/pkg/sqlstore/impl/system"
+	sqlstoreimpluser "github.com/textileio/go-tableland/pkg/sqlstore/impl/user"
+	"github.com/textileio/go-tableland/pkg/tables"
+	"github.com/textileio/go-tableland/pkg/tables/impl/ethereum"
+	"github.com/textileio/go-tableland/pkg/tables/impl/testutil"
+	"github.com/textileio/go-tableland/pkg/wallet"
+	"github.com/textileio/go-tableland/tests"
+)
+
+// ChainID is the test chain id.
+var ChainID = tableland.ChainID(1337)
+
+// FullStack holds all potentially useful components of the Tableland test stack.
+type FullStack struct {
+	Backend           *backends.SimulatedBackend
+	Address           common.Address
+	Contract          *ethereum.Contract
+	TransactOpts      *bind.TransactOpts
+	Sk                *ecdsa.PrivateKey
+	Wallet            *wallet.Wallet
+	TblContractClient *ethereum.Client
+	Client            *client.Client
+}
```

Comment:
I just tried to identify all the potentially important things a test may want access to. Me me know if these should change or names aren't clear.